pkg/api/job_request.go

```diff
@@ -13,6 +13,7 @@ import (
 type Container struct {
 	Name       string   `json:"name" yaml:"name"`
 	Image      string   `json:"image" yaml:"image"`
+	Devices    string   `json:"devices" yaml:"devices"`
```

Comment:
This is the job request that is sent out by Semaphore to the Agent. Currently the rest of the system does not support sending this out.

It seems that we are not using this field to construct the docker compose file in the implementation. Can we delete this line?

Reply:
Unit test fails if i delete this, just tested by removing just that line.

Reply:
Weird, it shouldn't be connected. Did we use this somewhere? 🤔

Reply:
Yes it's used in tests:
https://github.com/semaphoreci/agent/pull/83/files#diff-6a3b0052ae837341e9f6c4c682daf0c8R17

Reply:
Removed unnecessary additions. Please take another look.